README.md

```diff
@@ -1,22 +1,63 @@
-# What's Cookin'? Starter Kit 
+# What's Cookin' - Part One!
 
-The details of this project are outlined in the <a href="https://frontend.turing.edu/projects/What%27sCookin-PartOne.html" target="\__blank">project spec</a>.
+[Deploy Link](https://nzambonivergara.github.io/whats-cookin/)
 
-## Set Up 
+## Contributors
 
-1. Within your group, decide on one person to have the project repository on their Github account. This person will *fork* this repository - on the top right corner of the page, click the fork button. 
-2. All group members should then clone down the forked repository (make sure that everyone is added as a collaborator as well). Since you don't want your project to be named "whats-cookin-starter-kit", add an optional argument after the repo url when cloning. The command should look like this: `git clone [remote-address] [what you want to name the repo]`.
-3. Once you have cloned the repo, change into the directory and install the project dependencies. Run `npm install` or `npm i` to install project dependencies.
-4. Run `npm start` and visit `localhost:8080`
-5. Make sure both members of your team are collaborators on the forked repo.
+- [Natalia Zamboni Vergara](https://github.com/nzambonivergara)
+- [Paul Chambers](https://github.com/PaulTimothyChambers)
+- [Juliana Fargnoli](https://github.com/jfargnoli01)
 
-## Testing 
+## Abstract
 
-Mocha and chai are already set up, with a boilerplate test for you. 
+This application is a virtual recipe database where users can browse and save recipes for later use! The user is able to view all recipes, add and view their favorite recipes in the favorites section, as well as add and view recipes in the weekly recipes section. Users can use the live search bar at the top of the page to find their meal by name or ingredietns! If the recipe is no longer of use, the user can delete the recipe card from the favorites and or weekly views.
```

Comment:
"If the recipe is no longer of use, the user can delete the recipe card from the favorites >and/or< weekly views"